### DISCUSSION TITLE
how we make pdf of ipynb in vs code

### DISCUSSION THREAD

**Discussion by Saibtain** [AammarTufail/python_ka_chilla2023#53] [General]
how we make pdf of ipynb in vs code .

**Comment by nouman073** [comment]
Read the previous discussion.

**Reply by usamabajwa86** [reply]
install below mentioned Libraries in terminal window

1_ pip install nbconvert
2_pip install pyppeteer

**Reply by ZahidUllah1992** [reply]
pip install nbconvert
pip : The term 'pip' is not recognized as the name of a cmdlet, function, script file, or 
operable program. Check the spelling of the name, or if a path was included, verify that the 
path is correct and try again.
At line:1 char:1
+ pip install nbconvert

**Reply by ZahidUllah1992** [reply]
Still doesnot work
